### PR TITLE
Dues screen added

### PR DESCRIPTION
**Pull request by mahimak13**
@RitwikSingh28 Sir I have added dues screen page.  kindly review it. 
Screenshot 2023-10-29 133216

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -9,8 +11,8 @@ class MyApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return const MaterialApp(
-      home: LandingPage(),
+    return MaterialApp(
+      home: Scaffold(body: DuesScreen()),
```
Comment (RitwikSingh28) [inline lib/main.dart]: Please reset the `home` to `LandingPage`

**Review by RitwikSingh28 — CHANGES_REQUESTED** [review]
Good work @mahimak13
Please make the one change that I have mentioned in the preceding comment, resolve that and then we are good to go with your work

**Comment by mahimak13** [comment]
> Good work @mahimak13 Please make the one change that I have mentioned in the preceding comment, resolve that and then we are good to go with your work

Sir I have resolved the changes and have also added no dues page. Kindly review the changes
image

**Review by RitwikSingh28 — APPROVED** [review]
Great work @mahimak13

**Pull request merged**